Street.kt

```diff
@@ -0,0 +1,65 @@
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStreamReader
+import java.util.ArrayList
+
+class Street {
+    //get the capacity
+    val capacity = 8 //capacity of the street
+    private val countOfAllCars = 12 //all cars
+    private var drivingCars: Int = 0 //all cars, now driving (12 or less)
+    var allCars = ArrayList<Car>()
+
+    //contructor
+    init {
+        for (i in 0..countOfAllCars - 1) {
+            try { //Exception handling
+                allCars.add(Car())
+            } catch (e: Exception) {
+                //do something usefull or useless
+            }
+
+        }
+    }
+
+    //
+    fun countOfDrivingCars(): Int {
+        drivingCars = 0 //new loop (like next hour)
+        var driveIt: Boolean //drive the actual car from the list?
+        for (i in 0..countOfAllCars - 1) {
+            driveIt = allCars[i].isDriving()
+            if (driveIt) {
+                drivingCars ++
+            }
+        }
+        return drivingCars
+    }
+
+    companion object {
+
+        @JvmStatic fun main(args: Array<String>) {

```

Comment:
Wow, you're pulling out all the stops. Why not simply have a `Main.kt ` file with a `fun main` as in all the lectures?